### DISCUSSION TITLE
Server Actions

### DISCUSSION THREAD

**Discussion by Diwanshumidha** [community/community#68537] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Server Action is currently in Beta Phase so can we use it in Apps or should we stick to api

**Comment by MacMillon-VS** [comment]
Stick with API for now.

**Reply by Diwanshumidha** [reply]
Thats what i was thinking of

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
